recipes/picard/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "picard" %}
+{% set pypi_name = "python-picard" %}
+{% set version = "0.4" %}
+
+package:
+  name: {{ name|lower }}
```

Comment:
What do you think about making this the same as the  PyPI name? It seems more intuitive to me but I know nothing about either package.

Reply:
I don't really have a strong opinion on that; I've just changed it in aefb43a